src/components/ui/tooltip.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+
+import * as React from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+
+interface TooltipProps {
+  children: React.ReactNode;
+  tooltipContent: string;
+  tooltipTitle: string;
+}
+
+export function Tooltip({
+  children,
+  tooltipContent,
+  tooltipTitle,
+}: TooltipProps) {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const isTouchDevice = React.useRef(false);
+
+  React.useEffect(() => {
+    isTouchDevice.current =
+      'ontouchstart' in window || navigator.maxTouchPoints > 0;
+  }, []);
+
+  return (
+    <TooltipPrimitive.Provider>
+      <TooltipPrimitive.Root open={isOpen} onOpenChange={setIsOpen}>
+        <TooltipPrimitive.Trigger
+          asChild
+          onClick={(e) => {
+            if (isTouchDevice.current) {
```

Comment:
Does radix not properly support mobile?

I believe this condition will be true for devices that are not touch-only like windows surfaces, or really anything witha touchscreen even if it also has a regular mouse. Is that expectde?